tools/slienc.c

```diff
@@ -53,7 +53,7 @@ int main(int argc, const char **argv, const char **envp)
 	fseek(fp, 0, SEEK_SET);
 	
 	bz = malloc(insize);
-	fread(bz, 1, insize, fp);
+	size_t fread_result = fread(bz, 1, insize, fp);
```

Comment:
I'd argue this isn't the best way to fix the warning, but since it's tool code and not game code I don't think it matters too much.